partner_financial_risk/README.rst

```diff
@@ -0,0 +1,67 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+======================
+Partner Financial Risk
+======================
+
+Adds a new page in partner to manage the partner's *Financial Risk*
+If any limit is exceed the the partner is blocked to confirm sale orders.
```

Comment:
Hi @elicoidal , the main purpose of module it that the user (CFO or similar) could just set an amount of money that customer can use before paying it. But decision about this amount is analyzed outside Odoo. For Example a company which sale computer: CFO will decide that Customer1 will have 6.000 € of risk  just because this customer usually buy around that amount in computer every month. So if Customer1 wants to buy 12.000€ one month system will block this operation and CFO o similar should approve this. All operations bellow 6.000 will be approved automatically.
Conclusion: in this case Risk is just a field.